eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -12,9 +12,9 @@ function Invoke-AzBoardsCmd($subCmd, $parameters, $output = $true)
   return Invoke-Expression "$azCmdStr" | ConvertFrom-Json -AsHashTable
 }
 
-function LoginToAzureDevops([string]$devop_pat)
+function LoginToAzureDevops([string]$devops_pat)
 {
-  if (!$devop_pat) {
+  if (!$devops_pat) {
     return
```

Comment:
Should this be throwing an exception?

Reply:
No as there are a lot of usages where folks are already logged in locally when running the script so that user will be used. For example the cases where devs are running the new Prepare-Release scripts will be logged in as themselves when updating the release items. This pat option is only for our pipelines.